contrib/init/bitcoind.service

```diff
@@ -12,7 +12,7 @@ Description=Bitcoin daemon
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf -pid=/run/bitcoind/bitcoind.pid
+ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf -pid=/run/bitcoind/bitcoind.pid -printtoconsole -logtimestamps=0
```

Comment:
Don't combine `-daemon` and `-printtoconsole` - stdout will have nowhere to go after daemonization.

Reply:
The idea was to let systemd handle bitcoind's log output. At first using `-daemon` and `-printtoconsole` appeared to do the trick, but on a second look it sees that the log output stops after bitcoind demonized (possibly after the `fork()`?)

Reply:
> The idea was to let systemd handle bitcoind's log output. At first using -daemon and -printtoconsole appeared to do the trick, but on a second look it sees that the log output stops after bitcoind demonized (possibly after the fork()?) 

You could make work this by removing -daemon and -pid options, removing PIDFile setting, and changing Type setting from forking to simple. Personally, I would just want to use bitcoin logging, though, so logs can be more easily deleted if they contain private information, or if I enable verbose debugging and want to reclaim space. I would also leans towards using bitcoin logging here just to avoid adding unnecessary config overrides and differences between the way bitcoin functions inside and outside of systemd.

Reply:
I see your rationale, but on the other hand there are also good reasons to let systemd handle the log output of bitcoind. But since the this change was broken anyways (and since we can decide what's best later on,) I've reverted the commit with the hope that it would make it easier to get this PR merged.